src/FSharp.Control.AsyncSeq/FSharp.Control.AsyncSeq.fsproj

```diff
@@ -1,77 +1,23 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <SchemaVersion>2.0</SchemaVersion>
-    <ProjectGuid>2d4f3d0e-0e18-48d7-9c27-98864039393d</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <RootNamespace>FSharp.Control.AsyncSeq</RootNamespace>
-    <AssemblyName>FSharp.Control.AsyncSeq</AssemblyName>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
-    <TargetFSharpCoreVersion>4.3.0.0</TargetFSharpCoreVersion>
-    <Name>FSharp.Control.AsyncSeq</Name>
-    <TargetFrameworkProfile />
+    <TargetFrameworks>netstandard2.0;net45</TargetFrameworks>
+    <Version>2.0.14</Version>
```

Comment:
will be passed by fake, is `/p:Version` is used, so can be omitted (usually is less maintenance)